render.js

```diff
@@ -99,10 +99,8 @@ function object(o) {
 }
 
 function activeStateDecarator(stateRouter, element, stateName) {
-	var parametersToMatch = parseParameters(arguments)
-	function onStateChange(toState, toParams) {
-		var currentName = toState.name
-		var active = currentName.indexOf(stateName) === 0 && allParametersMatch(parametersToMatch, toParams)
+	function onStateChange() {
+		var active = stateRouter.stateIsActive(stateName)
```

Comment:
hmm, I think the parameters should still work when passed to the decorator - this should probably be `var active = stateRouter.stateIsActive(stateName, parametersToMatch)`